dev/sparktestsupport/modules.py

```diff
@@ -256,9 +256,21 @@ def __hash__(self):
 )
 
 
+mllib_local = Module(
+    name="mllib-local",
+    dependencies=[sketch],
```

Comment:
I moved the `SparkExpcetion` into `sketch` so we can use it in mllib-local. If we decide not to use Saprk specific exception or annotation in `mllib-local`, I can remove it.